### PR TITLE
benchmarks: optimize trie layer

### PR DESCRIPTION
**Pull request by aramikm**
# Goal
The goal of this PR is to optimize benchmark weights by isolating the **stateful-storage** to have a higher value for `additional-trie-layers` due to using of child trees instead of applying it to all pallets.

Related to #2093 

# Checklist
- [x] Benchmarks added?
- [x] Spec version incremented?

### CODE REVIEW

File: pallets/stateful-storage/src/weights.rs [hunk 1]
```diff
@@ -17,11 +17,11 @@
 // --chain=frequency-bench
 // --heap-pages=4096
 // --wasm-execution=compiled
-// --additional-trie-layers=5
 // --steps=20
 // --repeat=10
 // --output=./scripts/../pallets/stateful-storage/src/weights.rs
 // --template=./scripts/../.maintain/frame-weight-template.hbs
+// --additional-trie-layers=6
```
Comment (aramikm) [inline pallets/stateful-storage/src/weights.rs:24]: since we passed 1_000_000 I'm increasing this to 6

log<sub>16</sub>(1000000) ~ 5

**Review by saraswatpuneet — APPROVED** [review]
👍🏽

**Review by shannonwells — APPROVED** [review]
For posterity, I noted where I saw significant weight changes.
Looks good to go!

**Review by JoeCap08055 — APPROVED** [review]

**Review by enddynayn — APPROVED** [review]
👍 superb!

**Pull request merged**